This week's ST release changes (#8791)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/dbt-cloud-release-notes-gen.md

Release notes are grouped by date for single-tenant environments.

## March 11, 2026

## New

### Deployment and Configuration

- **Self-serve Snowflake private endpoint requests:** You can request a new Snowflake private endpoint from account settings by pasting the output from `SELECT SYSTEM$GET_PRIVATELINK_CONFIG();`, then track request status in the private endpoints table. This is available for Enterprise Business Critical accounts only, and please contact your account manager to enable. For other connection types, contact support@dbtlabs.com.

## Enhancements

### Orchestration and Run Status

> - **Run retries support dbt Fusion runs:** You can now retry failed runs as long as at least one configured dbt Core version is `1.6` or higher, including runs that used dbt Fusion.

Reviewer: This phrasing on this one is a little confusing. What does "at least one configured dbt Core version" mean? Should this be saying 
```suggestion
- **Run retries support dbt Fusion runs:** You can now retry failed runs as long as your environment is on dbt Core version `1.6` or higher or dbt Fusion.
```
